ztp/source-crs/StorageLVMCluster.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: odf-lvmcluster
+  namespace: openshift-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  storage:
+    deviceClasses:
+    - name: vg1
+      thinPoolConfig:
+        name: thin-pool-1
+        sizePercent: 90
+        overprovisionRatio: 10
```

Comment:
Array entries are hard to remove, so this entry forces users into using the name "vg1". One option would be to use:
```spec: {}```
That would force users to provide all the content (from storage: down) in their PolicyGenTemplate which seems appropriate for something as variable as storage configuration.

Reply:
Ok. Included a default configuration as a comment in the lvmcluster source-cr.  

So, a default PGT looks like this:

```yaml
apiVersion: ran.openshift.io/v1
kind: PolicyGenTemplate
metadata:
  name: "eric1"
  namespace: "ztp-eric1-policies"
spec:
  bindingRules:
    name: "eric1"
  mcp: "master"
  remediationAction: inform
  sourceFiles:
    - fileName: StorageLVMOSubscriptionNS.yaml
      policyName: "lvmo-policy"
    - fileName: StorageLVMOSubscriptionOperGroup.yaml
      policyName: "lvmo-policy"
    - fileName: StorageLVMOSubscription.yaml
      policyName: "lvmo-policy"
    - fileName: StorageLVMCluster.yaml
      policyName: "lvmo-config"
      spec:
        storage:
          deviceClasses:
          - name: vg1
            thinPoolConfig:
              name: thin-pool-1
              sizePercent: 90
              overprovisionRatio: 10
```

Notice that this config will create a VG with all the available devices except the disk where OCP is install. On top of that, thin-pool LVs will be created